app/models/competition.rb

```diff
@@ -2923,4 +2953,8 @@ def self.form_data_json_schema
       },
     }
   end
+
+  def auto_accept_threshold_reached?
+    auto_accept_disable_threshold > 0 && auto_accept_disable_threshold <= registrations.competing_status_accepted.count
+  end
```

Comment:
This method feels a bit "lonely" down here, can you find a better place for it? (Somewhere above among methods with a similar purpose perhaps)

Reply:
Moved it up to join the `registration_full?` and `registration_full_and_accepted?` methods, where it seems like the same theme sort of exists (asking questions about the state of the comp's registrations versus the competition's limits)